src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Validators/FileUpload/FileUploadStartViewModelValidator.cs

```diff
@@ -0,0 +1,97 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.ProviderCommitments.Configuration;
+using SFA.DAS.ProviderCommitments.Web.Models;
+using SFA.DAS.ProviderCommitments.Web.Models.Cohort;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Web.Validators
+{
+    public class FileUploadStartViewModelValidator : AbstractValidator<FileUploadStartViewModel>
+    {
+        private readonly ILogger<FileUploadStartViewModelValidator> _logger;
+        private readonly CsvConfiguration _csvConfiguration;
+
+        public FileUploadStartViewModelValidator(ILogger<FileUploadStartViewModelValidator> logger, CsvConfiguration csvConfiguration)
+        {
+            _logger = logger;
+            _csvConfiguration = csvConfiguration;
+
+            CascadeMode = CascadeMode.Stop;
+
+            RuleFor(x => x.Attachment)
+                .NotNull()
+                .Must(CheckFileSize).WithMessage("The selected file must be smaller than 50KB")
```

Comment:
this message should use string interpolation to include the csvConfiguration.MaxFileSize

Reply:
updated